src/main/java/org/codehaus/mojo/flatten/FlattenMojo.java

```diff
@@ -447,6 +451,67 @@ protected boolean shouldSkipGoal() {
         return skipFlatten;
     }
 
+    private void logParameters() {
+
+        getLog().info("Generating flattened POM of project " + this.project.getId() + " with following parameters "
+                + this.asString(this.getParameters()));
+    }
```

Comment:
This method and corresponding is not needed we have all this values in debug log, executing Maven with -X
like:
```
ehaus.mojo:flatten-maven-plugin:1.6.1-SNAPSHOT:flatten from plugin realm ClassRealm[plugin>org.codehaus.mojo:flatten-maven-plugin:1.6.1-SNAPSHOT, parent: jdk.internal.loader.ClassLoaders$AppClassLoader@63c12fb0]
[DEBUG] Configuring mojo execution 'org.codehaus.mojo:flatten-maven-plugin:1.6.1-SNAPSHOT:flatten:default-cli' with basic configurator -->
[DEBUG]   (f) defaultOperation = flatten
[DEBUG]   (f) embedBuildProfileDependencies = false
[DEBUG]   (f) flattenMode = bom
[DEBUG]   (f) flattenedPomFilename = .flattened-pom.xml
[DEBUG]   (f) keepCommentsInPom = false

```